posthog/__init__.py

```diff
@@ -423,3 +423,7 @@ def _proxy(method, *args, **kwargs):
 
     fn = getattr(default_client, method)
     return fn(*args, **kwargs)
+
+
+class Posthog(Client):
+    pass
```

Comment:
As it stands, I think the `disabled` setting wouldn't work, rest seem good!

Fine with keeping it like so, as long as we mention in the docs